Utils/hdinsight-node-common/pom.xml

```diff
@@ -232,6 +232,7 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure-mgmt-storage</artifactId>
+            <version>${azure.version}</version>
```

Comment:
why is this not needed before?

Reply:
I remove track1: azure-mgmt-storage from maven repository